code/modules/projectiles/guns/energy/laser.dm

```diff
@@ -623,8 +623,26 @@
 	weapon_weight = GUN_TWO_HAND_ONLY
 	can_scope = TRUE
 	init_firemodes = list(
-		/datum/firemode/semi_auto/slower
+		/datum/firemode/burst/two/slow
 	)
+
+/obj/item/gun/energy/laser/aer9/focused/rynn
+	name = "\improper Hot-wired AER10 laser rifle"
+	desc = "A pre-war laser rifle prototype that has seen better day, known as a failed attempt at competing with the wattz 2000. Part of it's grip seems broken along it's barrel, and it has cloth wrapped around it's stock and grip."
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/lasgun/hitscan/focused)
```

Comment:
Assuming you're trying to keep things condensed with your refractor, you don't need to state the ammotype, since it's a child of focused.